mmcv/ops/fused_bias_leakyrelu.py

```diff
@@ -45,9 +50,14 @@ def backward(ctx, gradgrad_input, gradgrad_bias):
         # The second order deviation, in fact, contains two parts, while the
         # the first part is zero. Thus, we direct consider the second part
         # which is similar with the first order deviation in implementation.
-        gradgrad_out = ext_module.fused_bias_leakyrelu(
-            gradgrad_input, gradgrad_bias.to(out.dtype), out, 3, 1,
-            ctx.negative_slope, ctx.scale)
+        gradgrad_out = ext_module.\
```

Comment:
The changing line here looks strange. Could we just use `ext_module.fused_bias_leakyrelu` in a single line?